app/models/user.rb

```diff
@@ -1,6 +1,17 @@
 class User < ActiveRecord::Base
-	validates :facebook_id, :name, :birthday, :image, presence: true
+	validates :facebook_id, :first_name, :last_name, :birthday, :image, presence: true
 	# validates :facebook_id, uniqueness: true
 	# has_many :tabs, :user_ratings
 	# validates_associated :tabs, :user_ratings
+  def self.by_location location
+    self.select { |u| u.most_recent_location == location }
+  end
+
+  def most_recent_location
+    # TODO. Location translator/geocoder/
+    # Need a model that will translate coordinates to cities,neighborhoods, etc
+    # ideally return something like this here:
+    # Locator.new.city location_history.pop
```

Comment:
i like this thinking. we can put this on the back burner until i figure out how to pass coordinates from the mobile app